manifests/t/Tencent/WeChat/3.9.5.90/Tencent.WeChat.locale.en-US.yaml

```diff
@@ -0,0 +1,40 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=QUSU.CRLF.7-3-5.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Tencent.WeChat
+PackageVersion: 3.9.5.90
+PackageLocale: en-US
+Publisher: 腾讯科技(深圳)有限公司
```

Comment:
```suggestion
Publisher: Tencent Technology (Shenzhen) Co., Ltd
```

Reply:
The `Publisher` field should match the ARP entry written by the installer. Although the installer can write different `DisplayName` values under different locales, it writes the same value for `Publisher` entry.
![image](https://github.com/microsoft/winget-pkgs/assets/56779163/7a9761fa-ad1b-4c95-9b5e-3ca350ebd0ee)